### ISSUE TITLE
in qlr map download, layers should be in a group

### ISSUE THREAD

**Issue by gubuntu** [kartoza/geonode#333]
Currently the qlr map download works and puts each layer at root level in the QGIS canvas. 

Add a tweak to put the layers into a QGIS group layer that has the name of the map.

 Might need more work on the otf-project-plugin

**Comment by Gustry** [comment]
Yes, it needs work on the QGIS Server plugin. Is-it a high priority or can we wait? It doesn't like a critical feature for now.

I need to do the qgis side and @boney-bun will do the geonode side.

**Comment by gubuntu** [comment]
low priority, it's still in backlog.

**Comment by gubuntu** [comment]
@Gustry any progress on this? In your absence who will be the best person to take on this and other otf-plugin work?
